### PR TITLE
docker version

### PR DESCRIPTION
**Pull request by smerle33**

### CODE REVIEW

File: jenkins-agent.pkr.hcl [hunk 1]
```diff
@@ -183,7 +188,25 @@ source "azure-arm" "base" {
   }
 }
 
+source "docker" "base" {
+  # only amd64 supported
+  image = "ubuntu:20.04"
+  # persist image on local docker engine
+  commit = true
+  # some label to stay compliant
```
Comment (lemeurherve) [inline jenkins-agent.pkr.hcl:196]:
```suggestion
  # some labels to stay compliant
```

Comment (lemeurherve) [reply jenkins-agent.pkr.hcl:196]: Commiting this suggestion should be enough to trigger the build as I've added you to a team with rights here.

**Comment by dduportal** [comment]
Converting to draft as it is work in progress. Incoming tasks:

- Add Docker-CE agent compliant to infra.ci (e.g. adding VM templates)
- Define the deployment process step (should be done at the end)

**Comment by smerle33** [comment]
superseeded by #160

**Pull request closed without merging**